package.json

```diff
@@ -55,7 +55,7 @@
     "gh-pages": "^0.12.0",
     "html-webpack-plugin": "^2.22.0",
     "isomorphic-style-loader": "^2.0.0",
-    "node-sass": "^3.8.0",
```

Comment:
Would using latest for node-sass be safe here?

Reply:
Yep. node-sass is deprecated so `latest` won't change & since we only build this when needed, we should be able to catch any bugs. I'll pin the version just in case. 

The reason we upgraded is because the previous version wouldn't build with our current version of node.